pkg/lib/account.go

```diff
@@ -108,94 +100,104 @@ func (r *Account) Validate() error {
 	return nil
 }
 
-func (r *Account) Read(scan Bai2Scanner, input string, lineNum int) (int, string, error) {
+func (r *Account) Read(scan *Bai2Scanner, isRead bool) error {
```

Comment:
What does `isRead` configure? The method is called `Read` already.

Reply:
Changed with Read(scan *Bai2Scanner, isExisted bool)

isExisted variable specify how to get line of scanner

```
for line := scan.ScanLine(isExisted); line != ""; line = scan.ScanLine(isExisted)
```
If isExisted is true, will return last line of scanner
If isExisted is false, will return new line of scanner


Read action will do on several layer like as file, group, account, detail 
isExisted is useful between read action of each layer

Reply:
Ah okay. I think `useCurrentLine bool` is a bit clearer.